src/query/storages/common/index/src/bloom_index.rs

```diff
@@ -262,17 +262,16 @@ impl BloomIndex {
                     }
                 }
                 _ => {
-                    let source_columns_iter = data_blocks_tobe_indexed.iter().map(|block| {
-                        let value = &block.get_by_offset(index).value;
-                        value.convert_to_full_column(field_type, block.num_rows())
-                    });
-                    let column = Column::concat_columns(source_columns_iter)?;
+                    let value = DataBlock::concat_columns(data_blocks_tobe_indexed, index)?;
```

Comment:
The `data_blocks_tobe_indexed` always have only one block, we can modify the argument as `&DataBlock` and don't need call `concat_columns` function.